src/Cli/Microsoft.DotNet.Cli.Utils/CommandArgument.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
```

Comment:
Agree with the parser issue. The parser used in SDK should be able to handle all these case without extra code. Either remove these code and adapter SDK's parser or move these code to Blazor project and make it "private" for Blazor project

Reply:
This is an issue I would block on

Reply:
I updated this to use the `System.CommandLine` package directly and was, indeed, able to remove a lot of the extra cruft here and in the tool itself.